Formula/skymaker.rb

```diff
@@ -0,0 +1,23 @@
+class Skymaker < Formula
+  desc "Generates fake astronomical images"
+  homepage "https://www.astromatic.net/software/skymaker"
+  url "http://www.astromatic.net/download/skymaker/skymaker-3.10.5.tar.gz"
```

Comment:
Can this use HTTPS?

Reply:
yes, sorry. I am commiting an updated version of the formula with https